config/default.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
   WEBSOCKET_PORT: process.env.WEBSOCKET_PORT || 8080,
   WEBSOCKET_PING_TIME: 15000,
   ZOKRATES_WORKER_HOST: process.env.ZOKRATES_WORKER_HOST || 'worker',
+  STOP_QUEUE_AFTER_INVALID_BLOCK: process.env.STOP_QUEUE_AFTER_INVALID_BLOCK === 'true',
```

Comment:
I think the default operation is STOP. Can we then change variable to NONSTOP functionality by default being false? If this variable is not defined, then we will stop queues. This will be easier when deploying so that i don't need to define this variable unless i want to test. Else, you are forcing me to include it once this PR is merged.